openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/flagencoders/WheelchairFlagEncoder.java

```diff
@@ -288,61 +414,52 @@ public int defineNodeBits(int index, int shift) {
         	
         	// foot=no, restricted, private
         	if (way.hasTag("foot", restrictedValues)) {
+                if (debugSkippedWays) LOGGER.warn("way skipped (pedestrian no, restricted or private): " + way.getId() + " Tags: " + way.getTags().toString());
         		return EncodingManager.Access.CAN_SKIP;
         	}
         	
-            // http://wiki.openstreetmap.org/wiki/DE:Key:sac_scale
-            String sacScale = way.getTag("sac_scale");
-            if (sacScale != null) {
-            	// even "hiking" is probably not possible for wheelchair user 
-                return EncodingManager.Access.CAN_SKIP;
-            }
-
             if (way.hasTag(KEY_SIDEWALK, usableSidewalkValues)) {
             	return EncodingManager.Access.WAY;
             }
             
             // Runge
-            if (way.hasTag(KEY_SIDEWALK, noSidewalkValues) && assumedWheelchairAccessibleHighways.contains(highwayValue))
-           		return EncodingManager.Access.CAN_SKIP;
+            if (way.hasTag(KEY_SIDEWALK, noSidewalkValues) && assumedWheelchairAccessibleHighways.contains(highwayValue)) {
+                if (debugSkippedWays) LOGGER.warn("way skipped (in assumedWheelchairAccessibleHighways list with no sidewalks): " + way.getId() + " Tags: " + way.getTags().toString());
+                return EncodingManager.Access.CAN_SKIP;
+            }
 
             // explicit motorroads are not usable
-            if (way.hasTag("motorroad", "yes"))
+            if (way.hasTag("motorroad", "yes")) {
+                if (debugSkippedWays) LOGGER.warn("way skipped (motorroad): " + way.getId() + " Tags: " + way.getTags().toString());
                 return EncodingManager.Access.CAN_SKIP;
+            }
 
             // do not get our feet wet, "yes" is already included above
-            if (isBlockFords() && (way.hasTag(KEY_HIGHWAY, "ford") || way.hasTag("ford")))
+            if (isBlockFords() && (way.hasTag(KEY_HIGHWAY, "ford") || way.hasTag("ford"))) {
+                if (debugSkippedWays) LOGGER.warn("way skipped (ford): " + way.getId() + " Tags: " + way.getTags().toString());
                 return EncodingManager.Access.CAN_SKIP;
-            
-            boolean bicycleOrHorseOnlyWay = (way.hasTag(KEY_BICYCLE, KEY_DESIGNATED) || way.hasTag(KEY_BICYCLE, KEY_OFFICIAL) || way.hasTag("horse", KEY_DESIGNATED) || way.hasTag("horse", KEY_OFFICIAL)) && !way.hasTag(restrictions, intendedValues);
-            if (bicycleOrHorseOnlyWay)
-                return EncodingManager.Access.CAN_SKIP;
-            
-            if (restrictedWheelchairHighways.contains(highwayValue)) {
-            	// In some countries bridleways cannot be travelled by anything other than a horse, so we should check if they have been explicitly allowed for foot or pedestrian
-                if (highwayValue.equals(KEY_BRIDLEWAY) && !(intendedValues.contains(way.getTag("foot", "no")) || intendedValues.contains(way.getTag(KEY_WHEELCHAIR, "no")))) {
```

Comment:
Not sure if we should keep this in the code or not as it is still something that is valid in that in some countries, you can't use bridleways unless you are riding a horse. I guess it is something that is up for discussion